src/test/rpc/AccountObjects_test.cpp

```diff
@@ -376,12 +579,29 @@ class AccountObjects_test : public beast::unit_test::suite
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::escrow), 0));
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::fee), 0));
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::hashes), 0));
+        BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::nft_page), 0));
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::offer), 0));
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::payment_channel), 0));
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::signer_list), 0));
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::state), 0));
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::ticket), 0));
 
+        // gw mints an NFT so we can find it.
+        uint256 const nftID{token::getNextID(env, gw, 0u, tfTransferable)};
+        env(token::mint(gw, 0u), txflags(tfTransferable));
+        env.close();
+        {
+            // Find the NFToken page and make sure it's the right one.
+            Json::Value const resp = acct_objs(gw, jss::nft_page);
+            BEAST_EXPECT(acct_objs_is_size(resp, 1));
+
+            auto const& nftPage = resp[jss::result][jss::account_objects][0u];
+            BEAST_EXPECT(nftPage[sfNFTokens.jsonName].size() == 1);
+            BEAST_EXPECT(
+                nftPage[sfNFTokens.jsonName][0u][sfNFToken.jsonName]
+                       [sfNFTokenID.jsonName] == to_string(nftID));
```

Comment:
@scottschurr just curious isnt `string` comparison more expensive than comparing in `uint256`?